tests/ypd/model/test_user.py

```diff
@@ -4,7 +4,8 @@
 from sqlalchemy import create_engine
```

Comment:
All of these tests fail, because we changed the way passwords are stored. You'll need to fix the tests